core/ErrorQueue.cc

```diff
@@ -147,6 +153,7 @@ vector<unique_ptr<core::ErrorQueueMessage>> ErrorQueue::drainFlushed() {
 }
 
 void ErrorQueue::markFileForFlushing(core::FileRef file) {
+    filesFlushedCount++;
```

Comment:
I think this does the right thing internally, but I prefer if you explicitly called the atomic version of `add`:

```suggestion
    filesFlushedCount.fetch_add(1);
```

That'll map to a CPU instruction for atomically incrementing an integer across CPU cores.